src/gondi.ts

```diff
@@ -318,6 +318,8 @@ export class Gondi {
         },
       };
     }
+
+    throw new Error(`Contract Address ${contractAddress} not supported`);
```

Comment:
I'd change the not supported message for something like `Invalid Contract Address ${address}`